plasmapy/diagnostics/thomson.py

```diff
@@ -31,7 +29,7 @@
     Te={"can_be_negative": False, "equivalencies": u.temperature_energy()},
     Ti={"can_be_negative": False, "equivalencies": u.temperature_energy()},
 )
-def spectral_density(
+def Maxwellian_spectral_density(
```

Comment:
After discussion with @rocco8773 , I think we're going to leave this as `spectral_density` for now. When other distribution functions are added we will refactor the module (@rocco8773 is opening an issue for this) and deal with the naming then. In the meantime, better to not break the existing naming.